docs/usage/self-hosted-configuration.md

```diff
@@ -363,6 +363,12 @@ Before the first commit in a repository, Renovate will:
 The `git` commands are run locally in the cloned repo instead of globally.
 This reduces the chance of unintended consequences with global Git configs on shared systems.
 
+## gitTimeout
+
+To handle the case where the underlying Git processes appear to hang, configure the timeout with the number of milliseconds to wait after last received content on either `stdOut` or `stdErr` streams before sending a `SIGINT` kill message.
+
+The value must be between `2000` and `6000` (milliseconds) inclusive.
```

Comment:
The default value is outside these values. How does that work?

Reply:
Its a typo in the text its should be `60000`